Nos/Views/AuthorStoryView.swift

```diff
@@ -251,9 +251,7 @@ fileprivate struct BottomOverlay: View {
         if replyCount == 0 {
             return nil
         }
-        let replyCount = replyCount
-        let localized = replyCount == 1 ? Localized.Reply.one : Localized.Reply.many
```

Comment:
This logic did not work for locales that support different pluralization rules than just `one` and `many`.